packages/sdks/snippets/angular-16-ssr/src/app/live-preview/live-preview.component.ts

```diff
@@ -0,0 +1,49 @@
+import { CommonModule } from '@angular/common';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import type { BuilderContent } from '@builder.io/sdk-angular';
+import { subscribeToEditor } from '@builder.io/sdk-angular';
+
+@Component({
+  standalone: true,
+  selector: 'app-live-preview',
+  imports: [CommonModule],
+  template: `
+    <div *ngIf="loading">Loading Data...</div>
+    <div *ngIf="!loading && content" class="blog-data-preview">
+      <div>Blog Title: {{ content.data?.title }}</div>
+      <div>Authored by: {{ content.data?.['author'] }}</div>
+      <div>Handle: {{ content.data?.['handle'] }}</div>
+    </div>
+  `,
+})
+export class LivePreviewComponent implements OnInit, OnDestroy {
+  content: BuilderContent | undefined = undefined;
+  loading = true;
+
+  private unsubscribeFn?: () => void;
+
+  constructor(private route: ActivatedRoute) {}
+
+  getFormattedDate(dateString: string | null): string {
+    return dateString ? new Date(dateString).toDateString() : '';
+  }
+
+  ngOnInit(): void {
+    this.content = this.route.snapshot.data['content'] || undefined;
```

Comment:
you shouldn't need the `|| undefined` for this

```suggestion
    this.content = this.route.snapshot.data['content'];
```